src/server/services/updateMeters.js

```diff
@@ -14,9 +15,19 @@ async function updateAllMeters() {
 	const time = new Date();
 	console.log(`Getting meter data ${time.toISOString()}`); // eslint-disable-line no-console
 	try {
-		const meters = await Meter.getAll();
-		// Do all the network requests in parallel.
-		const readingInsertBatches = await Promise.all(meters.filter(m => m.enabled && m.type === Meter.type.MAMAC).map(readMamacData));
+		const allMeters = await Meter.getAll();
+		const metersToUpdate = allMeters.filter(m => m.enabled && m.type === Meter.type.MAMAC);
+
+		// Do all the network requests in parallel, then throw out any requests that fail after logging the errors.
+		const readingInsertBatches = _.filter(await Promise.all(
+			metersToUpdate
+				.map(readMamacData)
+				.map(p => p.catch(err => {
+					console.error(err);
```

Comment:
Instead of printing out the entire error, could we only print out failing meter id and the RequestError.message, i.e. "Error: connect ETIMEDOUT 8.8.8.8:80" if ip address 8.8.8.8 was failing.

So it would look like: Error: connect ETIMEDOUT 8.8.8.8:80 from meter id 14